src/components/views/elements/Pill.js

```diff
@@ -29,7 +29,7 @@ const REGEX_MATRIXTO = new RegExp(MATRIXTO_URL_PATTERN);
 
 // For URLs of matrix.to links in the timeline which have been reformatted by
 // HttpUtils transformTags to relative links. This excludes event URLs (with `[^\/]*`)
-const REGEX_LOCAL_MATRIXTO = /^#\/(?:user|room|group)\/(([#!@+])[^\/]*)$/;
+const REGEX_LOCAL_MATRIXTO = /^#\/(?:user|room|group)\/(([#!@+])[^/]*)$/;
```

Comment:
This also doesn't seem right... I think...

Reply:
According to https://regex101.com/r/1M2H7B/2/

`\/ matches the character / literally (case sensitive)`

and 

`/ matches the character / literally (case sensitive)`

so they are the same